### PR TITLE
PE-48 | Implemented Camera Form to Allow Users to Add Pictures 

### PR DESCRIPTION
**Pull request by tonynguyen98**
**What I Did;**
- Used the [Image Picker Package](https://pub.dev/packages/image_picker) to allow users to add images along their "Create a Question Post".
- Used [Photo View Package](https://pub.dev/packages/photo_view) to allow users to open their image and zoom and preview it.
- Used Firebase Storage to upload all images that the user wants and then return the download link of that image.

**How to Test:**
1. I've added a couple of packages, so you will need to run `flutter pub get`.
2. Open the emulator or physical phone and take some pictures to add into the phone's gallery.
3. Create a new question with the button on the bottom of home screen. 
4. The title and description can be what every you want.
5. Then switch to the `Photo Tab` and choose the option to `Take a New Photo`.
6. The screen should update now that an image is added.
7. Take add a couple more pictures but the `Pick Photo from Gallery` option.
8. Then more pictures however you want. 
9. Go back to the `Text` tab and click submit. You then should see a screen saying it's being uploaded.
10. When upload complete, you should see a confirmation of the title, description, and images on the bottom.
11. In you web browser, go to Firebase Storage and you should see the image uploaded there.

### CODE REVIEW

File: lib/components/eureka_camera_form.dart [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        borderRadius: BorderRadius.circular(15.0),
+        child: Image.file(
+          File(
+            widget.mediaPaths[index],
+          ),
+          fit: BoxFit.cover,
+        ),
+      ),
+    );
+  }
```
Comment (MaxLian11) [inline lib/components/eureka_camera_form.dart:147]: When clicking on the image it shows an error "`Failed assertion: boolean expression must not be null`"

**Review by MaxLian11 — CHANGES_REQUESTED** [review]
Code looks very clean and image uploading works great. I also like the UI of New Question page. However there is one bug that I found when opening uploaded picture in full screen.
I also have a question, after the question is submitted and clicked return to home page, it goes to rating screen, is that temporarily for demo purposes?

**Review by ayeejane — APPROVED** [review]
Amazing job! Everything works as it should.

**Review by MaxLian11 — APPROVED** [review]
Works as intended. Excellent job

**Review by davidsquines — APPROVED** [review]
Works as intended

**Pull request merged**